webapp/pages/post/_id/_slug/index.vue

```diff
@@ -90,7 +90,17 @@
           @toggleNewCommentForm="toggleNewCommentForm"
         />
         <ds-space margin-bottom="large" />
-        <hc-comment-form v-if="showNewCommentForm" :post="post" @createComment="createComment" />
+        <comment-form
+          v-if="showNewCommentForm && !post.author.isBlocked"
+          :post="post"
+          @createComment="createComment"
+        />
+        <ds-placeholder v-else>
+          {{ $t('settings.blocked-users.explanation.commenting-disabled') }}
+          <br />
+          {{ $t('settings.blocked-users.explanation.commenting-explanation') }}
+          <a href="https://support.human-connection.org/kb/">FAQ</a>
+        </ds-placeholder>
```

Comment:
@mattwr18, you've been asking about the placeholder a few times – of course I'm not extremely happy about it since we're trying to get rid of styleguide components when we can. But in this case I think a) it looks fine, b) I don't know a better (and reusable!) solution off the top of my head and c) we'll have to migrate the `ds-placeholder` anyway, so we can leave it for future-us. :grin:

Reply:
ok, I looked at in the styleguide and it's quite simple, but we can look at it, and design it later